app/views/support_interface/single_provider_user_removals/new.html.erb

```diff
@@ -0,0 +1,13 @@
+<span class="govuk-caption-xl"><%= @user.display_name %></span>
```

Comment:
can we also add a back link, just to inline with all our other designs?

Reply:
good call, done